content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -318,8 +318,9 @@ same as the scheduler.
 A Pod can restart, causing re-execution of Init Containers, for the following
 reasons:
 
-* A user updates the PodSpec causing the Init Container image to restart the entire pod.
-  App Container image changes only restart the app Container.
+* A user updates the PodSpec causing the Init Container image to change. Any
+  changes to the Init Container image restarts the Pod.App Container image 
```

Comment:
@SupriyaSirbi  nit: Please add a (space) after .....Init Container image restarts the Pod.

Reply:
ok done